### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by ElsLommelen**
This PR solves 2 unrelated small problems:
- add dependency dbplyr which is used in the inboveg functions
- change driver to the current Windows standard

Could you please check the latter by trying function `connect_inbo_dbase()`?

### CODE REVIEW

**Comment by florisvdh** [comment]
@ElsLommelen regarding your question https://github.com/inbo/tutorials/pull/205#issuecomment-716642137, I have looked at your driver-filtering code in this PR. I believe your new take for Windows should be robust enough to apply for various Linux installations, so I believe you can indeed safely drop the previously existing separate approach for Linux in this PR. :+1: 

``` r
library(odbc)
driversvec <- unique(odbcListDrivers()$name)
drivers_sql <- driversvec[grepl("SQL Server", driversvec)]
drivers_sql_odbc <-
    drivers_sql[grepl("ODBC Driver", drivers_sql)]
tail(sort(drivers_sql_odbc), 1)
#> [1] "ODBC Driver 17 for SQL Server"
```

<sup>Created on 2020-10-26 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value                       
#>  version  R version 4.0.3 (2020-10-10)
#>  os       Linux Mint 20               
#>  system   x86_64, linux-gnu           
#>  ui       X11                         
#>  language nl_BE:nl                    
#>  collate  nl_BE.UTF-8                 
#>  ctype    nl_BE.UTF-8                 
#>  tz       Europe/Brussels             
#>  date     2020-10-26                  
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version date       lib source        
#>  assertthat    0.2.1   2019-03-21 [1] CRAN (R 4.0.2)
#>  backports     1.1.10  2020-09-15 [1] CRAN (R 4.0.2)
#>  bit           4.0.4   2020-08-04 [1] CRAN (R 4.0.2)
#>  bit64         4.0.5   2020-08-30 [1] CRAN (R 4.0.2)
#>  blob          1.2.1   2020-01-20 [1] CRAN (R 4.0.2)
#>  callr         3.5.1   2020-10-13 [1] CRAN (R 4.0.3)
#>  cli           2.1.0   2020-10-12 [1] CRAN (R 4.0.3)
#>  crayon        1.3.4   2017-09-16 [1] CRAN (R 4.0.2)
#>  DBI           1.1.0   2019-12-15 [1] CRAN (R 4.0.2)
#>  desc          1.2.0   2018-05-01 [1] CRAN (R 4.0.2)
#>  devtools      2.3.2   2020-09-18 [1] CRAN (R 4.0.2)
#>  digest        0.6.26  2020-10-17 [1] CRAN (R 4.0.3)
#>  ellipsis      0.3.1   2020-05-15 [1] CRAN (R 4.0.2)
#>  evaluate      0.14    2019-05-28 [1] CRAN (R 4.0.2)
#>  fansi         0.4.1   2020-01-08 [1] CRAN (R 4.0.2)
#>  fs            1.5.0   2020-07-31 [1] CRAN (R 4.0.2)
#>  glue          1.4.2   2020-08-27 [1] CRAN (R 4.0.2)
#>  highr         0.8     2019-03-20 [1] CRAN (R 4.0.2)
#>  hms           0.5.3   2020-01-08 [1] CRAN (R 4.0.2)
#>  htmltools     0.5.0   2020-06-16 [1] CRAN (R 4.0.2)
#>  knitr         1.30    2020-09-22 [1] CRAN (R 4.0.2)
#>  magrittr      1.5     2014-11-22 [1] CRAN (R 4.0.2)
#>  memoise       1.1.0   2017-04-21 [1] CRAN (R 4.0.2)
#>  odbc        * 1.2.3   2020-06-18 [1] CRAN (R 4.0.2)
#>  pkgbuild      1.1.0   2020-07-13 [1] CRAN (R 4.0.2)
#>  pkgconfig     2.0.3   2019-09-22 [1] CRAN (R 4.0.2)
#>  pkgload       1.1.0   2020-05-29 [1] CRAN (R 4.0.2)
#>  prettyunits   1.1.1   2020-01-24 [1] CRAN (R 4.0.2)
#>  processx      3.4.4   2020-09-03 [1] CRAN (R 4.0.2)
#>  ps            1.4.0   2020-10-07 [1] CRAN (R 4.0.3)
#>  R6            2.4.1   2019-11-12 [1] CRAN (R 4.0.2)
#>  Rcpp          1.0.5   2020-07-06 [1] CRAN (R 4.0.2)
#>  remotes       2.2.0   2020-07-21 [1] CRAN (R 4.0.2)
#>  rlang         0.4.8   2020-10-08 [1] CRAN (R 4.0.3)
#>  rmarkdown     2.4     2020-09-30 [1] CRAN (R 4.0.3)
#>  rprojroot     1.3-2   2018-01-03 [1] CRAN (R 4.0.2)
#>  sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 4.0.2)
#>  stringi       1.5.3   2020-09-09 [1] CRAN (R 4.0.2)
#>  stringr       1.4.0   2019-02-10 [1] CRAN (R 4.0.2)
#>  testthat      2.3.2   2020-03-02 [1] CRAN (R 4.0.2)
#>  usethis       1.6.3   2020-09-17 [1] CRAN (R 4.0.2)
#>  vctrs         0.3.4   2020-08-29 [1] CRAN (R 4.0.2)
#>  withr         2.3.0   2020-09-22 [1] CRAN (R 4.0.2)
#>  xfun          0.18    2020-09-29 [1] CRAN (R 4.0.2)
#>  yaml          2.2.1   2020-02-01 [1] CRAN (R 4.0.2)
#> 
#> [1] /home/floris/lib/R/library
#> [2] /usr/local/lib/R/site-library
#> [3] /usr/lib/R/site-library
#> [4] /usr/lib/R/library
```

</details>

**Comment by ElsLommelen** [comment]
Ok, should be really ready for review now!  ;-)

File: R/connect_inbo_dbase.R [hunk 1]
```diff
@@ -46,15 +46,12 @@ connect_inbo_dbase <- function(database_name, autoconvert_utf8 = TRUE) {
         type <- "INBO PRD Server"
     }
 
-    sql_driver <- if (.Platform$OS.type == "unix") {
```
Comment (florisvdh) [inline R/connect_inbo_dbase.R:49]: you've lost the critical `sql_driver <-` part here

**Review by florisvdh — CHANGES_REQUESTED** [review]

File: R/connect_inbo_dbase.R (hunk 1 above)
Comment (ElsLommelen) [reply R/connect_inbo_dbase.R:49]: Woops...

**Review by florisvdh — APPROVED** [review]

**Review by hansvancalster — APPROVED** [review]
Nice! It works like a charm on my Windows laptop.
A few things that need to be changed:

- add yourself as author
- remove unused functions from importFrom (I thing dbGetQuery and dbListTables
- update documentation files

**Comment by ElsLommelen** [comment]
Thanks @florisvdh and @hansvancalster !
I met your comments, and as the new odbc version is already on CRAN in the meantime, I added the suggestion to install the latest odbc version in the error message of the recently solved error.  (In the long term we can just remove this whole adapted function, but I would just leave it like this for some time to avoid questions.)

**Pull request merged**